man/systemd-analyze.xml

```diff
@@ -792,6 +792,314 @@ Service b@0.service not loaded, b.socket cannot be started.
         as well and its default value is 100.</para></listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><option>--security-policy=<replaceable>PATH</replaceable></option></term>
+
+        <listitem><para>With <command>security</command>, allow the user to define a custom set of
+        requirements formatted as a JSON file against which to compare the specified unit file(s)
+        and determine their overall exposure level to security threats.</para></listitem>
+      </varlistentry>
+
+      <table>
+          <title>Accepted Directives</title>
```

Comment:
not sure "directives" is the right word. Maybe "Assessment Test Identifiers" or so?